### PR TITLE
add description field

### LINKED ISSUE #81: add description field in room creation

### PR DESCRIPTION
**Pull request by HlainePoeEi**
Resolve #81

### CODE REVIEW

File: frontend/src/components/Room/CreateRoom.js [hunk 1]
```diff
@@ -107,13 +112,24 @@ const CreateRoom = () => {
...
                                 },
                             ]}
                         >
                             <Input placeholder="Enter room name" />
                         </Form.Item>
 
+                        <Form.Item
+                            name="description"
+                            label="Description :"
+                            id="description"
```
Comment (myonaingwinn) [inline frontend/src/components/Room/CreateRoom.js:125]:
```suggestion
                            id="description"
```

Is this **id** needed?

**Comment by myonaingwinn** [comment]
image
two user count?

File: frontend/src/components/Room/CreateRoom.js (hunk 1 above)
Comment (HlainePoeEi) [reply frontend/src/components/Room/CreateRoom.js:125]: ma htae ll ya tal. ma use voo

Comment (HlainePoeEi) [reply frontend/src/components/Room/CreateRoom.js:125]: > Is this **id** needed?

ma htae ll ya tal. ma use voo

**Comment by HlainePoeEi** [comment]
> image
> 
> 1. two user count?
> 2. no description layout, with description layout (should be)
> 3. action buttons alignment and spacing

1. user count ka bar mha ma pyin htar voo lay

**Comment by myonaingwinn** [comment]
image
* can u **customize** _validation error 

- [ ] messages_

such as : 
- [ ] Room name is required.
- [ ] Description cannot be longer than 100 characters.

**Comment by myonaingwinn** [comment]
@HlainePoeEi 
お疲れ様でした。有難うございます。

**Pull request merged**